pr_agent/agent/pr_agent.py

```diff
@@ -8,6 +8,13 @@
 from pr_agent.tools.pr_reviewer import PRReviewer
 
 
+def addfsda(a, asd,sad):
```

Comment:
Remove the added function. It does not seem to be related to the project's functionality and contains harmful code. [important]